warehouse/query-core/src/test/java/datawave/query/util/WiseGuysIngest.java

```diff
@@ -1055,7 +1055,7 @@ private static Value getValueForBuilderFor(String... in) {
      */
     private static Value getValueForNuthinAndYourHitsForFree() {
         Uid.List.Builder builder = Uid.List.newBuilder();
-        builder.setCOUNT(50); // better not be zero!!!!
+        builder.setCOUNT(0); // better not be zero!!!!
```

Comment:
I was tempted to force you to fix this comment, but I already did that in the future versions ;)

Reply:
I was trying to keep the same commit hash!